src/Controls/src/Core/Handlers/Items/ItemsViewHandler.Android.cs

```diff
@@ -31,6 +33,32 @@ protected override void DisconnectHandler(RecyclerView platformView)
 		protected override RecyclerView CreatePlatformView() =>
 			new MauiRecyclerView<TItemsView, ItemsViewAdapter<TItemsView, IItemsViewSource>, IItemsViewSource>(Context, GetItemsLayout, CreateAdapter);
 
+		public override Size GetDesiredSize(double widthConstraint, double heightConstraint)
+		{
+			var width = widthConstraint;
+			var height = heightConstraint;
+
+			if (!double.IsInfinity(width))
+				width = Context.ToPixels(width);
+
+			if (!double.IsInfinity(height))
+				height = Context.ToPixels(height);
+
+			UpdateEmptyViewSize(width, height);
```

Comment:
If the ItemsView measurement is unconstrained in one direction, won't that set the height or width of the EmptyViewAdapter's RecyclerViewWidth or RecyclerViewHeight to infinity? It seems like it should use the height returned from GetDesiredSize, not the measurement constraints.